Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js

```diff
@@ -895,6 +896,83 @@ YUI.add('ez-navigationhubviewservice-tests', function (Y) {
         },
     });
 
+    navigateToFirstItemTest = new Y.Test.Case({
+        name: "eZ Navigation Hub View Service navigate to first item tests",
+
+        setUp: function () {
+            var that = this;
+
+            this.zoneName = "midfield";
+
+            this.routeName = "Lass";
+            this.routeParams = [];
+            this.item = new Y.Base();
+
+            this.item.set('route', {
+                name: this.routeName,
+                params: this.routeParams
+            });
+
+            this.parameterItem = {
+                config: {
+                    route: {
+                        name: this.routeName,
+                        params: this.routeParams,
+                    }
+                }
+            };
+
+            this.appMock = new Mock();
+
+            Mock.expect(this.appMock, {
+                method: 'navigateTo',
+                args: [Mock.Value.String, Mock.Value.Any],
+                run: function (routeName, routeParams) {
+                    Assert.areSame(
+                        that.routeName,
+                        routeName,
+                        "Route Name should have been provided"
+                    );
+                    Assert.areSame(
+                        that.routeParams,
+                        routeParams,
+                        "Route Params should have been provided"
+                    );
+                }
+            });
+
+            this.service = new Y.eZ.NavigationHubViewService({app: this.appMock});
+
+            this.view = new Y.View({
+                'bubbleTargets': this.service,
+                'activeNavigation': ''
+            });
+        },
+
+        tearDown: function () {
+            this.service.destroy();
+            delete this.service;
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should not do anything when no item": function () {
+            this.view.set('activeNavigation', this.zoneName);
```

Comment:
while the code appears to be covered by that, this is incomplete. Before this line, you should (re)configure `appMock` to not expect any call to `navigateTo` and then call `Mock.verify` at the end.